src/main/java/frc/robot/Constants.java

```diff
@@ -71,18 +81,62 @@
             public static final int kSpinnerMotorPort = 9; // motor for the wheel that spins the control panal
 
             public static final int kColorSensorPort = 0; // Color sensor that senses the colors on the control panel
+
+            /*values the color sensor values to detect the colors on control panel
+            red*/
+            public static final double kRedTargetR = 0.476;
+            public static final double kRedTargetG = 0.376;
+            public static final double kRedTargetB = 0.15;
+            //Yellow
+            public static final double kYellowTargetR = 0.381;
+            public static final double kYellowTargetG = 0.545;
+            public static final double kYellowTargetB = 0.136;
+            //Green
+            public static final double kGreenTargetR = 0.18;
+            public static final double kGreenTargetG = 0.568;
+            public static final double kGreenTargetB = 0.249;
+            //Blue
+            public static final double kBlueTargetR = 0.15;
+            public static final double kBlueTargetG = 0.4467;
+            public static final double kBlueTargetB = 0.40;
+            //Colors
+            public static final Color kRedTarget = new Color(kRedTargetR, kRedTargetG, kRedTargetB);
+            public static final Color kYellowTarget = new Color(kYellowTargetR, kYellowTargetG, kYellowTargetB);
+            public static final Color kGreenTarget = new Color(kBlueTargetR, kBlueTargetG, kBlueTargetB);
+            public static final Color kBlueTarget = new Color(kGreenTargetR, kGreenTargetG, kGreenTargetB);
+
         }
     
-        public static final class HangerConstants {
+        public static final class ClimberConstants {
     
-            public static final int kRightHangerMotorPort = 2;
-            public static final int kLeftHangerMotorPort = 1;
+            public static final int kRightClimberMotorPort = 2;
+            public static final int kLeftClimberMotorPort = 1;
 
             public static final int kFrictionSolenoidPort = 6; // piston that applies the friction brake on the hanging elevator
         }
     
         public static final class LimeLightConstants {
-    
+           public static final double kLimeLightOffest = 13.67;
```

Comment:
I would suggest fixing the indentation in this file.

In VSCode you can do this by:
- Shortcut `Shift` + `Alt` + `F` on Windows **OR**
- `Ctrl` + `Shift` + `P` and type format